src/main/java/com/cs56fitnessapp/models/FoodEntry.java

```diff
@@ -0,0 +1,94 @@
+package com.cs56fitnessapp.models;
+
+public class FoodEntry {
+    private double servingSize;
+    private int calories;
+    private TotalFat totalFat;
+    private double cholesterol;
+    private double sodium;
+    private TotalCarb totalCarbs;
+    private double protein;
+    private double vitaminA;
+    private double vitaminC;
+    private double calcium;
+    private double iron;
+    public FoodEntry(double servingSize, int calories, TotalFat totalFat, double cholesterol, double sodium, TotalCarb totalCarbs, double protein, double vitaminA, double vitaminC, double calcium, double iron){
```

Comment:
Let's change the constructor to accepts only two params: 
`double servingSize, int calories` since these would be the only required onces. I added corresponding changes in the UML. So do not set other properties in the constructor. 

On top of the constructor please add a bold comment like
    /**
     * Constructs a FoodEntry object with serving size and calories per serving size
     * @param servingSize - serving size of the product
     * @param calories - calories per serving size
     */